openfisca_dummy_country/parameters/param_root.xml

```diff
@@ -18,34 +26,35 @@
       <BAREME code="bareme" description="Bareme progressif de contribution sociale sur les salaires">
         <TRANCHE code="tranche0">
           <SEUIL>
-            <VALUE deb="2013-01-01" fuzzy="true" valeur="0" />
+            <VALUE deb="2013-01-01" valeur="0" />
           </SEUIL>
           <TAUX>
-            <VALUE deb="2017-01-01" fuzzy="true" valeur="0.02" />
-            <VALUE deb="2015-01-01" fin="2016-12-31" valeur="0.04" />
-            <VALUE deb="2013-01-01" fin="2014-12-31" valeur="0.03" />
+            <VALUE deb="2016-01-01" valeur="0.02" />
```

Comment:
Same here, this example is used by other packages for tests. Why change `2017` to `2016` ?